test/unit/utils/test-constants.ts

```diff
@@ -0,0 +1,65 @@
+const area = {
+    name: "Kiambu, Kenya",
+    application: "gfw",
+    geostore: "33b01a49bf9b56a8b56ce042a24f6567",
+    wdpaid: null,
+    userid: "testuser",
+    createdAt: "2025-01-29T16:28:54.271Z",
+    updatedAt: "2025-01-29T16:28:54.271Z",
+    image: "",
+    datasets: [],
+    user: {},
+    env: "production",
+    tags: [],
+    status: "saved",
+    public: true,
+    fireAerts: true,
+    deforestationAlerts: true,
+    deforestationAlertsType: "glad-all",
+    webhookUrl: "",
+    monthlySummary: false,
+    subscriptionId: "testsub",
+    email: "test.user@wri.org",
+    language: "en",
+    confirmed: false
+}
+
+const admin2Iso = {
+    country: "KEN",
+    region: '15',
+    subregion: '1',
+    source: {
+        provider: "gadm",
+        version: "3.6"
+    }
+}
+
+const admin2Admin = {
+    adm0: "KEN",
+    adm1: '15',
+    adm2: '1',
```

Comment:
Technically, these are _Number_ types in the [Areas V2 Model](https://github.com/gfw-api/gfw-area/blob/production/app/src/models/area.modelV2.js#L61-L62). I'm not sure if this changes the behavior of the code or not.